_api-reference/ingest-apis/simulate-ingest.md

```diff
@@ -36,48 +34,54 @@ POST /_ingest/pipeline/35678/_simulate
 ```
 {% include copy-curl.html %}
 
-## Path and HTTP methods
+## Example requests
 
-Simulate the last ingest pipeline created.
+Requests to simulate the lastest ingest pipeline created:
 
 ```
 GET _ingest/pipeline/_simulate
 POST _ingest/pipeline/_simulate
 ```
 
-Simulate a single pipeline based on the pipeline's ID.
+Requests to simulate a single pipeline based on the pipeline identifier.
 
 ```
-GET _ingest/pipeline/{id}/_simulate
-POST _ingest/pipeline/{id}/_simulate
+GET _ingest/pipeline/pipeline-id/_simulate
+POST _ingest/pipeline/pipeline-id/_simulate
 ```
 
-## URL parameters
+## Path parameters
+
+Parameter | Type | Description
+:--- | :--- | :---
+`pipeline` | String | Pipeline identifier or wildcard expression of pipeline identifiers used to limit the request. 
```

Comment:
Wildcard does not work here. I feel like the previous format is better than having a separate `path parameters` section.